tests/upgrade/basic/task.yaml

```diff
@@ -18,7 +18,7 @@ execute: |
     echo Do upgrade
     # allow-downgrades prevents errors when new versions hit the archive, for instance,
     # trying to install 2.11ubuntu1 over 2.11+0.16.04
-    apt install -y --allow-downgrades ${SPREAD_PATH}/../snapd*.deb
+    dpkg -i ${SPREAD_PATH}/../snapd*.deb || apt-get -f install -y
```

Comment:
I mean, won't apt-get then decide the easiest way to fix any issue is to re-upgrade snapd?